### PR TITLE
Move form errors to redux store

### PR DESCRIPTION
**Pull request by shaundillon**
This is based on [Reeta's PR](https://github.com/guardian/media-atom-maker/pull/344/files) in the Media atom maker. Moving our form errors state out of the form scope and into the redux store,
in an object sorted by a form title.

/cc @clloyd

### CODE REVIEW

File: public/js/reducers/formErrorsReducer.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export default function formErrors(state = {}, action) {
+
+  let formName, newFormErrors, currentFormErrors, updatedFormErrors, updatedForm;
```
Comment (clloyd) [inline public/js/reducers/formErrorsReducer.js]: let?

Comment (shaundillon) [inline public/js/reducers/formErrorsReducer.js]: I just copied these from Reeta's haha. Will change.

Comment (clloyd) [inline public/js/reducers/formErrorsReducer.js]: I was more meaning `const`?

**Comment by clloyd** [comment]
@shaundillon I was referring to making them immutable, vars/lets in reducers make me a bit  nervous

**Pull request merged**